components/kubeflow-pipeline/python/pyproject.toml

```diff
@@ -12,8 +12,8 @@ readme = "README.md"
 packages = [
     { include = "fl_suite" }
 ]
+# Explicitly included as 'poetry' will exclude files in '.gitignore' by default
```

Comment:
Without this change, the `poetry install` step breaks as shown [here](https://github.com/katulu-io/fl-suite/runs/7728703967?check_suite_focus=true#step:9:39):

```
  RuntimeError

  The Poetry configuration is invalid:
    - [include.0] None is not valid under any of the given schemas
  

  at /opt/poetry/venv/lib/python3.9/site-packages/poetry/core/factory.py:43 in create_poetry
       39│             message = ""
       40│             for error in check_result["errors"]:
       41│                 message += "  - {}\n".format(error)
       42│ 
    →  43│             raise RuntimeError("The Poetry configuration is invalid:\n" + message)
       44│ 
       45│         # Load package
       46│         name = local_config["name"]
       47│         version = local_config["version"]
```

It is not clear to me as to why this is the case: the difference between `1.1.13` and `1.1.14` doesn't seem to have any relevant change: https://github.com/python-poetry/poetry/compare/1.1.13...1.1.14